cmd/create/createNode.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright © 2020 The k3d Author(s)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+package create
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	k3dc "github.com/rancher/k3d/pkg/cluster"
+	"github.com/rancher/k3d/pkg/runtimes"
+	k3d "github.com/rancher/k3d/pkg/types"
+	"github.com/rancher/k3d/version"
+	log "github.com/sirupsen/logrus"
+)
+
+// NewCmdCreateNode returns a new cobra command
+func NewCmdCreateNode() *cobra.Command {
+
+	// create new command
+	cmd := &cobra.Command{
+		Use:   "node NAME",
```

Comment:
should this be `node CLUSTER_NAME NAME` since `-c` is required and not really a flag

Reply:
That or make `-c` optional and default to `k3s-default`

Reply:
You're right, I cannot really think of a good use case for having standalone nodes here :thinking: 
I'd go for the second option of keeping the `-c` flag and making `k3s-default` the default value.